data/matching/fashion/popular_items.py

```diff
@@ -1,5 +1,5 @@
 from numpy import dtype
-from .type import Matcher
```

Comment:
why did you start using relative imports in some places? At most places we have absolute import.

Reply:
we can use relative import inside the modules (directories). makes our code more portable. the individual modules don't need to know what's the structure of the overall project, only what's inside their folder.
I have been using relative imports within modules for the last 6 months, successfully!